### PR TITLE
gh-93096: Remove `python -m base64 -t`

### PR DESCRIPTION
**Pull request by arhadthedev**
`test_base64.LegacyBase64TestCase.test_encodebytes`/`test_decodebytes` is a more advanced version of the removed CLI argument.

The removed feature brings no benefit to an end user (unlike the rest of `python -m base64`).

* Issue: gh-93096

### CODE REVIEW

File: Lib/test/test_base64.py [hunk 1]
```diff
@@ -762,14 +762,6 @@ def tearDown(self):
     def get_output(self, *args):
         return script_helper.assert_python_ok('-m', 'base64', *args).out
 
-    def test_encode_decode(self):
```
Comment (corona10) [inline Lib/test/test_base64.py:765]: I would like to add test sets into `test_encodebytes` and `test_encodebytes`

**Review by corona10 — CHANGES_REQUESTED** [review]

File: Lib/base64.py [hunk 2]
```diff
@@ -570,8 +570,7 @@ def main():
     usage = """usage: %s [-h|-d|-e|-u|-t] [file|-]
```
Comment (corona10) [inline Lib/base64.py]: nit: Can we update `usage` to use f-string?

Comment (arhadthedev) [reply Lib/base64.py]: Done. Should I go further and convert it to argparse?

**Comment by arhadthedev** [comment]
I have made the requested changes; please review again.

**Review by corona10 — APPROVED** [review]
lgtm

**Pull request merged**